llpc/tool/amdllpc.cpp

```diff
@@ -968,6 +973,7 @@ static Result buildPipeline(ICompiler *compiler, CompileInfo *compileInfo) {
       pipelineInfo->iaState.patchControlPoints = 3;
 
     pipelineInfo->options.robustBufferAccess = RobustBufferAccess;
+    pipelineInfo->options.robustAccessEx = RobustAccessEx;
```

Comment:
This always overwrites the value from the .pipe file, which must be wrong. Do we really need a command-line option anyway?

Reply:
I think we should follow the way of robustBufferAccess. After all, robustAccess is an enhanced version.

Reply:
Yes, looks like RobustBufferAccess is handled incorrectly too. Using amdllpc with no options on a dumped pipe with robust access will disable the robust access, so we lose the idea that using amdllpc on a dumped pipe should give the same result (pretty much) as it did in the driver.

Reply:
If you want to keep the options, then you could use getNumOccurrences (I might have got the name wrong) on the option to see if it was specified. If not specified, then leave the value from the pipe file.

Reply:
Considering this is incorrect usage, I think it's better to remove robustAccessEx in amdllpc.